app/views/components/_navbar.html.erb

```diff
@@ -12,19 +12,19 @@
     <ul class="navbar-nav mr-auto">
       <% if user_signed_in? %>
         <li class="nav-item active">
-          <%= link_to "Home", "#", class: "nav-link" %>
+          <%= link_to "List a garden", "/gardens/new", class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
           <%= image_tag "mary.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-            <%= link_to "Action", "#", class: "dropdown-item" %>
-            <%= link_to "Another action", "#", class: "dropdown-item" %>
+            <%= link_to "My Gardens", "#", class: "dropdown-item" %>
+            <%= link_to "Profile", "#", class: "dropdown-item" %>
             <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
           </div>
         </li>
       <% else %>
         <li class="nav-item">
-          <%= link_to "Login", new_user_session_path, class: "nav-link" %>
+          <%= link_to "Login or Sign up", new_user_session_path, class: "btn btn-light" %>
```

Comment:
Could add sign up tab as well as login (I added in my branch)
<%= link_to "Sign Up", new_user_registration_path, class: "nav-link" %>